src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you can wtite like this instead:
background: url(../images/star.svg) no-repeat center;